### PR TITLE
Mm-38333 display users within a channel

### PR DESCRIPTION
**Pull request by hamzaMM**
#### Summary
Added users within a group to list view in add members

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-38333

#### Related Pull Requests
NA

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/channel_invite_modal/channel_invite_modal.tsx [hunk 1]
```diff
@@ -97,13 +100,18 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         this.props.actions.getProfilesNotInChannel(this.props.channel.team_id, this.props.channel.id, this.props.channel.group_constrained, 0).then(() => {
             this.setUsersLoadingState(false);
         });
+        this.props.actions.getProfilesInChannel(this.props.channel.id, 0, USERS_PER_PAGE, '', true);
         this.props.actions.getTeamStats(this.props.channel.team_id);
         this.props.actions.loadStatusesForProfilesList(this.props.profilesNotInCurrentChannel);
+        this.props.actions.loadStatusesForProfilesList(this.props.profilesInCurrentChannel);
```
Comment (BenCookie95) [inline components/channel_invite_modal/channel_invite_modal.tsx:106]: Can we concat the 2 arrays (this.props.profilesInCurrentChannel & this.props.profilesNotInCurrentChannel) and make this into one request?

File: components/channel_invite_modal/channel_invite_modal.tsx [hunk 2]
```diff
@@ -97,13 +100,18 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
...
+
     }
 
     public onHide = (): void => {
         this.setState({show: false});
         this.props.actions.loadStatusesForProfilesList(this.props.profilesNotInCurrentChannel);
+        this.props.actions.loadStatusesForProfilesList(this.props.profilesInCurrentChannel);
```
Comment (BenCookie95) [inline components/channel_invite_modal/channel_invite_modal.tsx:113]: Can we concat the 2 arrays (this.props.profilesInCurrentChannel & this.props.profilesNotInCurrentChannel) and make this into one request?

**Review by BenCookie95 — CHANGES_REQUESTED** [review]
Left a few small comments, can we also remove the package-lock.json changes?

**Review by BenCookie95 — APPROVED** [review]

**Comment by BenCookie95** [comment]
@mkraft Unsure who to add as QA because Malik is on PTO

**Comment by esethna** [comment]
Hi @hamzaMM, are you keen on finishing this PR?

**Comment by furqanmlk** [comment]
@BenCookie95 can you please fix the conflict ?

**Review by furqanmlk — APPROVED** [review]
Working as expected.

**Before**

image

**After**

image

**Comment by esethna** [comment]
Awesome, lets get this merged! @BenCookie95 can you help and merge when possible?

**Pull request merged**